src/models/Collectible.js

```diff
@@ -57,9 +57,6 @@ export type CollectibleTransaction = {
   _id: string,
 };
 
-
-export type Collectibles = Collectible[];
-
 export type CollectiblesStore = {
```

Comment:
Maybe we can proceed with general renaming to `NFT` including types and etc.? As if someone else steps in they might continue on `collectible` narrative naturally. Wdyt?